collections.go

```diff
@@ -782,7 +783,11 @@ func handleViewCollection(app *App, w http.ResponseWriter, r *http.Request) erro
 
 	// Add more data
 	// TODO: fix this mess of collections inside collections
-	displayPage.PinnedPosts, _ = app.db.GetPinnedPosts(coll.CollectionObj)
+	if isOwner {
+		displayPage.PinnedPosts, _ = app.db.GetPinnedPosts(coll.CollectionObj, true)
+	} else {
+		displayPage.PinnedPosts, _ = app.db.GetPinnedPosts(coll.CollectionObj, false)
```

Comment:
We can simplify this kind of logic back to one line, and just use `isOwner` for the second parameter.